src/Runner.php

```diff
@@ -136,6 +139,12 @@ public function start() {
 			Logger::error( 'Failed to push changes to origin.' );
 		}
 
+		if ( $existing_PR_branch ) {
+			// TODO: Add comment to existing PR with $message
```

Comment:
Do you want to resolve this TODO?

Reply:
Unfortunately, hub does not have a 'comment' command. I started a pull request (https://github.com/github/hub/pull/1465), but there has been no feedback, so I'm not sure that it's wanted. We could use the curl API.